### PR TITLE
chromeStorageApi, storageService unit tests

### PR DESCRIPTION
**Pull request by m0t0r**
I have fixed all issues and took into account comments which were presented in the first pull request. 
This pull request contains:
- chromeStorageApi wrapper with unit tests
- storageService refactoring and unit tests

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -74,8 +74,8 @@
 <script src="scripts/controllers/storageunits.js"></script>
 <script src="scripts/controllers/batches.js"></script>
 <script src="scripts/controllers/bundle.js"></script>
-<script src="scripts/services/chromeStorageApi.js"></script>
-<script src="scripts/services/storageService.js"></script>
+<script src="scripts/services/chrome-storage-api.js"></script>
+<script src="scripts/services/storage-service.js"></script>
```
Comment (jjal) [inline app/index.html:79]: :+1:

**Comment by jjal** [comment]
- please rebase and re-issue
- new tests look good but fail locally for me - dependencies
- please update bower / karma conf with your sinon stuff

**Comment by m0t0r** [comment]
I have fixed issues. All unit tests have been passed. I think you can finally merge with develop. :)

Also, I removed siononJS from my test. So, seems we don't need it, jasmine spies work fine.

**Comment by tlvince** [comment]
To avoid superfluous merge commits, next time, you could use `git pull --rebase origin/develop`.

File: test/mock/chrome-storage-api.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+describe('chromeStorageApi', function () {
+  var mockWindow, chromeStorageApi;
+
+  beforeEach(module('lmisChromeApp'));
+
+  beforeEach(function () {
+    mockWindow = {
```
Comment (tlvince) [inline test/mock/chrome-storage-api.js]: With regards to my previous comment about moving the spec in to the `/mock` directory, I meant just the `mockwindow` object, not the spec itself. This should be moved back into the `/spec/services` directory. Sorry for the confusion.

**Comment by jjal** [comment]
When I merge this locally I get 

```
TypeError: Object.keys called on non-object
 at Function.keys (native)
at chrome-extension://.../scripts/services/storage-service.js:145:30
```

which is in getAll.- there is a test for that function but for some reason it doesn't catch this.
It's blocking saves of stock count so we'll need to figure that out before merging it in.

File: test/mock/chrome-storage-api.js (hunk 2 above)
Comment (m0t0r) [inline test/mock/chrome-storage-api.js]: Alright. I will fix it

**Pull request merged**